go/vt/orchestrator/logic/tablet_discovery.go

```diff
@@ -174,7 +178,7 @@ func LockShard(instanceKey inst.InstanceKey) (func(*error), error) {
 	if err != nil {
 		return nil, err
 	}
-	ctx, cancel := context.WithTimeout(context.TODO(), 1*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
we usually use topo.RemoteOperationTimeout for topo calls. That defaults to 30 seconds. Is that a reasonable default here (instead of 1 second)? Or do we want to introduce a new vitess flag / orchestrator config param?

Reply:
In this case, we don't want to wait that long. If a `LockShard` fails, it means somebody else has the lock. So, we should fail fast.